src/main/java/com/listywave/list/application/domain/category/CategoryType.java

```diff
@@ -11,19 +11,20 @@
 @AllArgsConstructor
 public enum CategoryType {
 
-    ENTIRE("0", "전체"),
-    CULTURE("1", "문화"),
-    LIFE("2", "일상생활"),
-    PLACE("3", "장소"),
-    MUSIC("4", "음악"),
-    MOVIE_DRAMA("5", "영화/드라마"),
-    BOOK("6", "도서"),
-    ANIMAL_PLANT("7", "동식물"),
-    ETC("8", "기타"),
+    ENTIRE("0", "전체", "https://image.listywave.com/category/category_entire.png"),
+    CULTURE("1", "문화", "https://image.listywave.com/category/category_culture.png"),
+    LIFE("2", "일상생활", "https://image.listywave.com/category/category_life.png"),
+    PLACE("3", "장소", "https://image.listywave.com/category/category_place.png"),
+    MUSIC("4", "음악", "https://image.listywave.com/category/category_music.png"),
+    MOVIE_DRAMA("5", "영화/드라마", "https://image.listywave.com/category/category_movie_drama.png"),
+    BOOK("6", "도서", "https://image.listywave.com/category/category_book.png"),
+    ANIMAL_PLANT("7", "동식물", "https://image.listywave.com/category/category_animal_plant.png"),
+    ETC("8", "기타", "https://image.listywave.com/category/category_etc.png"),
     ;
 
     private final String codeValue;
     private final String korNameValue;
+    private final String categoryImageUrl;
```

Comment:
```suggestion
    private final String code;
    private final String viewName;
    private final String imageUrl;
```
로 변수명 바꾸는 거 어떠신가요????